### ISSUE TITLE
RunInScript inspection has a false positive on annotation + method call with lambdas

### ISSUE THREAD

**Issue by akuleshov7** [analysis-dev/diktat#1272]
The following code in `*.kts` scripts is refactored by RunInScript inspection:

```
@Suppress("")
publishJar {}
```

to:

```
run {
   @Suppress("")
   publishJar {}
}
```

This is incorrect and it breaks the logic in gradle scripts. Annotations should be handled separately.
